### ISSUE TITLE
Add ability to record Jitsi conference

### ISSUE THREAD

**Issue by akontsevich** [vector-im/element-meta#1564]
### Description
Please add ability to record conference in Jitsi plugin

### Version information
- **Platform**: web (in-browser) and desktop
- **OS**: Windows, macOS, Ubuntu, Linux, etc

**Comment by akontsevich** [comment]
Yes adding recording and Youtube live stream features to Jitsi integration plugin for Riot would be great!!! Killing feature for Skype replacement! With screensharing as well of course: vector-im/element-desktop#683. Waiting for implementation. 

Also good to have ability to unfold Jitsi widget to whole chat area or even whole Riot window! Created ticket for this: vector-im/element-web#8851.

**Comment by hex-m** [comment]
As a server operator who is able to set up a Jitsi Meet Instance for their homeserver this is already possible with Jibri as mentioned above. Such a setup was used for FOSDEM 2021 and you can read about it [here](https://matrix.org/blog/2021/02/15/how-we-hosted-fosdem-2021-on-matrix).

The plan seems to be to implement Matrix-native group calling instead of the Jitsi-Meet widget.
